.circleci/config.yml

```diff
@@ -0,0 +1,73 @@
+version: 2
+
+defaults: &defaults
+  working_directory: ~/go/src/github.com/docker/libnetwork
+  docker:
+    - image: 'circleci/golang:1.10'
+      environment:
+          Dockerfile: Dockerfile.ci
+          dockerargs:  --privileged -e CIRCLECI
```

Comment:
not sure if here we need privileged

Reply:
if we dont run container without privileged option, lot of testing which invoke system calls failed. I already came across during my testing